### PR TITLE
FA-111, FA-117 Runnable App with Navigation Added.

### PR DESCRIPTION
**Pull request by brosscode**
App is hooked up, launchable, and runnable with a java configuration.

### CODE REVIEW

File: appJava/src/main/AndroidManifest.xml [hunk 1]
```diff
@@ -14,6 +14,13 @@
         android:supportsRtl="true"
         android:theme="@style/Theme.ForzenBook.DayNight"
         tools:targetApi="31">
+        <activity
+            android:name=".LandingPageActivity"
+            android:exported="false">
+            <meta-data
```
Comment (ForzenAcademy) [inline appJava/src/main/AndroidManifest.xml]: I don't think you need this

File: appJava/src/main/java/com/brandon/appjava/di/NavigationModule.java [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+import dagger.hilt.InstallIn;
+import dagger.hilt.android.components.ActivityComponent;
+import dagger.hilt.android.qualifiers.ActivityContext;
+
+@Module
+@InstallIn(ActivityComponent.class)
+public class NavigationModule {
+
+    @Provides
+    public LegacyNavigationJava providesNavigation(@ActivityContext Context context) {
```
Comment (ForzenAcademy) [inline appJava/src/main/java/com/brandon/appjava/di/NavigationModule.java]: Make the function calls do this, not the navigator

File: appLegacy/src/main/res/values/strings.xml [hunk 3]
```diff
@@ -0,0 +1 @@
+<resources></resources>
```
Comment (ForzenAcademy) [inline appLegacy/src/main/res/values/strings.xml]: Remove file

File: loginModuleJava/src/main/java/com/brandon/loginmodulejava/view/LoginActivity.java [hunk 4]
```diff
@@ -110,9 +111,9 @@ this, getString(R.string.core_error_no_internet_connection),
                             binding.loginEmailError.setVisibility(View.GONE);
                         }
                         if (it.isCodeSent()) {
-                            binding.buttonText.setText(com.brandon.uicore.R.string.core_get_code_text);
```
Comment (ForzenAcademy) [inline loginModuleJava/src/main/java/com/brandon/loginmodulejava/view/LoginActivity.java:113]: Import R?

File: appJava/src/main/AndroidManifest.xml (hunk 1 above)
Comment (brosscode) [reply appJava/src/main/AndroidManifest.xml]: Removed.

File: appLegacy/src/main/res/values/strings.xml (hunk 3 above)
Comment (brosscode) [reply appLegacy/src/main/res/values/strings.xml]: Removed.

File: appJava/src/main/java/com/brandon/appjava/di/NavigationModule.java (hunk 2 above)
Comment (brosscode) [reply appJava/src/main/java/com/brandon/appjava/di/NavigationModule.java]: Fixed Memory Leak

File: loginModuleJava/src/main/java/com/brandon/loginmodulejava/view/LoginActivity.java (hunk 4 above)
Comment (brosscode) [reply loginModuleJava/src/main/java/com/brandon/loginmodulejava/view/LoginActivity.java:113]: Fixed import issue

**Review by ForzenAcademy — APPROVED** [review]

**Pull request merged**